libs/auth/src/apdu.ts

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable max-classes-per-file */
+import { Buffer } from 'buffer';
+import { Byte, isByte } from '@votingworks/types';
+import { assert } from '@votingworks/basics';
+
+/**
+ * The max length of an APDU
```

Comment:
I describe APDUs more fully (including what the abbreviation stands for :smile:) in the comment for this file's main exported class and entry point, `CommandApdu`